pdsfile/pds4file/__init__.py

```diff
@@ -0,0 +1,98 @@
+################################################################################
+# pds4file subpackage & Pds4File subclass with PdsFile as the parent class
+################################################################################
+
+from pdsfile import *
```

Comment:
Need to avoid import *. There should be no need for this.

Reply:
Fixed